Content.Server/Voting/VoteCommands.cs

```diff
@@ -129,6 +179,85 @@ public CompletionResult GetCompletion(IConsoleShell shell, string[] args)
             var n = args.Length - 1;
             return CompletionResult.FromHint(Loc.GetString("cmd-customvote-arg-option-n", ("n", n)));
         }
+
+        // Sends the payload's message. 
+        private async void WebhookMessage(WebhookPayload payload, string id)
+        {
+            if (id == string.Empty)
+            {
+                var request = await _httpClient.PostAsync($"{_webhookUrl}?wait=true",
```

Comment:
Is `?wait=true` necessary here?

Reply:
Yes, it won't wait for a response otherwise